tenets/codelingo/code-review-comments/declare-empty-slice/codelingo.yaml

```diff
@@ -0,0 +1,34 @@
+funcs:
+  - name: fixSliceType
+    type: resolver
+    body: |
+      function(sliceType) {
+        sliceType = sliceType.split(' ').join('');
+        return  sliceType.substring(0, sliceType.indexOf("{"))
+      }
+
+tenets:
+  - name:  declare-empty-slice
+    flows:
+      codelingo/review:
+        comment: Declare {{sliceName}} as a nil slice, with `var {{sliceName}} {{fixSliceType(sliceType)}} [as specified in Code Review Comments](https://github.com/golang/go/wiki/CodeReviewComments#declaring-empty-slices)
+      # A rewrite only works with raw for now. See generated query for this example https://www.codelingo.io/playground/?id=37. 
+      # Also, if a struct is defined in one file and a variable of the struct type is declared in another, we get "invalid type" in the generated query
```

Comment:
@leilaes oh ok, awesome! Nice to see a rewrite tenet! That is slightly out of scope, because we’re more concerned about finding issues that top of funnel - but I’m sure that @mullikine will be pleased. I know this one isn't perfect, and we should probably either account for the "invalid type" issue, but it's good enough to put back in the top of funnel I'd say!